projects/imagemagick/encoder_fuzzer.cc

```diff
@@ -0,0 +1,24 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+#define FUZZ_ENCODER_STRING_LITERAL(name) #name
+#define FUZZ_ENCODER FUZZ_ENCODER_STRING_LITERAL(FUZZ_IMAGEMAGICK_ENCODER)
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  const Magick::Blob blob(Data, Size);
+  Magick::Image image;
+  try {
+    image.read(blob);
```

Comment:
I've tried running it locally and got a memory leak almost instantly. It seems that memory is allocated when throwing an exception, but not released after that. Can we do anything inside the `catch` block to properly de-allocate the resources used by an exception object?

Reply:
A good question. http://www.imagemagick.org/Magick++/Exception.html doesn't appear to think it's necessary to deallocate manually so maybe this is a bug or maybe @dlemstra can tell us what to do?

Reply:
Another quick ping now that the holidays are over, @dlemstra is there something we're missing around exception memory here?

Reply:
Interesting, I can't reproduce it either now. I guess I might've used some unlucky revision previously, but current ToT works well for me.

Reply:
Great!